client/src/components/Header.js

```diff
@@ -39,21 +74,15 @@ export default ({ currentLanguage, languageChangeCallback }) => {
           </NavLink>
         </li>
       </ul>
-      <div
-        className="languageChangeButton"
-        role="button"
-        onClick={() => {
-          if (currentLanguage === 'en') {
-            languageChangeCallback('ro');
-            setLangText('English');
-          } else {
-            languageChangeCallback('en');
-            setLangText('Romană');
-          }
-        }}
+      <Dropdown
+        overlay={() => languageMenu(langText, handleLanguageBtnClick)}
+        trigger={['hover', 'click']}
```

Comment:
I think you can keep only the `click` behaviour for the language switcher. Testing the new language switcher I saw that I end up moving the mouse over it, click and the language drop-down disappears and I have to click again.

Reply:
Behaviour sample:
https://user-images.githubusercontent.com/2820084/107880087-43439200-6ee5-11eb-964a-3df763888e31.mov